src/shared/popups/gokb-edit-platform-popup/gokb-edit-platform-popup.vue

```diff
@@ -0,0 +1,230 @@
+  >
+    <gokb-error-component :value="error" />
+    <span v-if="successMsg">
+      <v-alert
+        type="success"
+        dismissible
+      >
+        {{ localSuccessMessage }}
+      </v-alert>
+    </span>
+    <span v-if="errorMsg">
+      <v-alert
+        type="error"
+        dismissible
+      >
+        {{ localErrorMessage }}
+      </v-alert>
+    </span>
+    <v-row>
+      <v-col md="12">
+        <template>
+          <div v-if="isEdit">
+            <v-row dense>
+              <v-col cols="6">
+                <gokb-text-field
+                  v-model="localAction"
+                  :label="$t('component.platform.edit')"
+                  disabled
+                />
+              </v-col>
+            </v-row>
+          </div>
+          <div v-else>
+            <v-row dense>
+              <v-col cols="6">
+                <gokb-text-field
+                  v-model="localAction"
+                  :label="$t('component.platform.create')"
+                  disabled
+                />
+              </v-col>
+            </v-row>
+          </div>
+        </template>
+      </v-col>
+    </v-row>
+    <v-row
+      dense
+      class="mt-4"
+    >
+      <v-col>
+        <gokb-title-field
+          v-model="platform.title"
+          :label="$tc('component.platform.label')"
+          required
+          dense
+          return-object
+        />
+      </v-col>
+    </v-row>
+    <v-row dense>
+      <v-col>
+        <gokb-url-field
+          ref="platformUrl"
+          v-model="platform.url"
+          :label="$tc('component.tipp.url.label')"
+          required
+          dense
+        />
+      </v-col>
+    </v-row>
+
+    <template #buttons>
+      <v-spacer />
+      <gokb-button
+        @click="close"
+      >
+        {{ $t('btn.cancel') }}
+      </gokb-button>
+      <gokb-button
+        v-if="!isReadonly"
+        :disabled="!isValid"
+        default
+      >
+        {{ isEdit ? $t('btn.update') : $t('btn.create') }}
+      </gokb-button>
+    </template>
+  </gokb-dialog>
+</template>
+
+<script>
+  import BaseComponent from '@/shared/components/base-component'
+  import accountModel from '@/shared/models/account-model'
+  import platformServices from '@/shared/services/platform-services'
+  import 'vue-json-pretty/lib/styles.css'
+
+  export default {
+    name: 'GokbEditPlatformPopup',
+    extends: BaseComponent,
+    props: {
+      selected: {
+        type: Object,
+        required: false,
+        default: undefined
+      },
+      readonly: {
+        type: Boolean,
+        required: false,
+        default: false
+      },
+      component: {
+        type: Object,
+        required: false,
+        default: undefined
+      }
+    },
+    data () {
+      return {
+        selectedItem: undefined,
+        successMsg: undefined,
+        errorMsg: undefined,
+        updateUrl: undefined,
+        platform: {
+          label: undefined,
+          url: undefined
```

Comment:
The platform property in the backend is called `primaryUrl`. It is okay to remap this, but we have to keep track of this (on retrieval & on save).